core/src/main/java/io/questdb/cutlass/line/LineTcpSender.java

```diff
@@ -74,12 +78,36 @@ public static LineTcpSender newSender(int ip, int port, int bufferCapacity) {
         }
     }
 
+    @Override
+    public final void at(long timestamp, ChronoUnit unit) {
+        put(' ').put(timestamp * unit.getDuration().toNanos() / 1000).put("tus");
```

Comment:
this is insane:

![image](https://github.com/questdb/questdb/assets/7276403/495a77ad-5020-4dd6-8d60-7584133db8c0)

Could we check the sender latency before and after pls?